src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1006,6 +1045,32 @@ public final void testPostPatronAccountByIdInstanceByInstanceIdHold() {
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdInstanceByInstanceIdHoldPatronComments() {
```

Comment:
Is there already an existing test that creates a hold request for a patron?

Reply:
Yes such a test exists, will leverage it.